internal/core/graphics/color.rs

```diff
@@ -416,6 +416,21 @@ pub(crate) mod ffi {
     #![allow(unsafe_code)]
     use super::*;
 
+    #[no_mangle]
+    pub unsafe extern "C" fn slint_color_red(col: &Color, out: u8) {
```

Comment:
I think it's overkill to expose the function like that.  The C++ code generator can just use the existing `to_argb_*` function.

Reply:
This was me flailing around trying to figure out where what how. Looks like I have everything sorted now.